src/PortingAssistant.Client/PortingAssistantCLI.cs

```diff
@@ -118,6 +128,32 @@ public void HandleCommand(String[] args)
                     {
                         PortingProjects = o.PortingProjects.ToList();
                     }
+
+                    switch (o.MinimumLoggingLevel)
```

Comment:
Let's make this case insensitive by making `MinimumLoggingLevel` lower case.

Reply:
Fixed!